README.md

```diff
@@ -10,8 +10,13 @@ PySTAC is a library for working with [SpatialTemporal Asset Catalog](https://sta
 
 ## Installation
 
-PySTAC requires Python>=3.6. Support for Python>=3.10 should be considered experimental
-until further notice.
+PySTAC requires Python>=3.7. This project follows the recommendations of
```

Comment:
Might be more readable with spaces: `Python >= 3.7`

Reply:
Yeah, good call.

Reply:
Fixed in c6bf9510e82aebcf800ef73769856dbde34f12ba